Wireframe/css/style.css

```diff
@@ -191,20 +191,6 @@ footer p {
   font-size: 0.8em;
```

Comment:
Don't make font tiny. Most people cannot read small fonts. We make websites for other people to use.

Reply:
I’ve adjusted the font sizes to make sure they’re more readable and accessible for all users. Thanks for the reminder to keep usability in mind!

Please let me know if you think the new sizes work better.